<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellTypeDlg</class>
 <widget class="QDialog" name="CellTypeDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Please define cell types</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTableWidget" name="cellTypeTable">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="baseSize">
      <size>
       <width>256</width>
       <height>171</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>Cell Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Freeze</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Cell Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="cellTypeLE"/>
       </item>
       <item>
        <widget class="QCheckBox" name="freezeCHB">
         <property name="toolTip">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Indicates whether cells of this type should remain frozen during simulation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Freeze</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cellTypeAddPB">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="clearCellTypeTablePB">
         <property name="text">
          <string>Clear Table</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>cellTypeTable</zorder>
  <zorder>clearCellTypeTablePB</zorder>
  <zorder>label_4</zorder>
  <zorder>cellTypeAddPB</zorder>
  <zorder>cellTypeLE</zorder>
  <zorder>freezeCHB</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder></zorder>
  <zorder>okPB</zorder>
  <zorder>cancelPB</zorder>
  <zorder>horizontalSpacer_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okPB</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelPB</sender>
   <signal>clicked()</signal>
   <receiver>CellTypeDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
